### ISSUE TITLE
Failed to generate 

### ISSUE THREAD

**Issue by ashleycroft** [IntelOrca/biorand#742]
### Version

3.1.8 (2f583cd)

### Seed

R710-P5L1-XXHH40M539YNH07JT1H82414ZZZZZZZZZZZZZZZZZZZZZZZZZ

### Scenario

RE 1 (Chris)

### Player Character

Leon (RE6) Jill (RE3)

### What happened?

I'm trying to generate RE1 but it keeps failing. I can generate RE2 just fine and i got biorand as admin. It also says integrity check failed saying directors cut camera angles are not supported. Rename your bio2dc to something else.

**Comment by IntelOrca** [comment]
Yes, you need to rename your bio2dc folder. Alternatively just download the bundle off biorand.net.

**Comment by ashleycroft** [comment]
The bio1dc is the one i need to rename not bio2dc. What you recommend
renaming the bio1dc folder too?

**Comment by IntelOrca** [comment]
Anything you like, `_bio1dc` is what I normally do.

**Comment by ashleycroft** [comment]
When i click generate it keeps saying failed to generate. Here's what it
says in a small box "Biorand does not have permission to write to this
location. If your game is installed in 'Program Files', you may need to run
BioRand as  administrator. I already have Biorand to run as admin do you
know what i'm doing wrong? I installed the seamless hd mod on all games do
i need to remove that from all 3 games for the biorand to generate?

**Comment by IntelOrca** [comment]
I recommend downloading the bundle from biorand.net.

One of the issues people hit is that the install folder is marked read only.

**Comment by ashleycroft** [comment]
I already downloaded the bundle from Biorand.net. There are 3 options to
choose from which one do i want to download? I downloaded the first option
that just says Biorand. The second one says Resident Evil + Biorand and the
third one says Resident Evil + HD pack + Biorand.

**Comment by ashleycroft** [comment]
Do i need to download Resident Evil 1 PC (Original Uncensored)?

**Comment by IntelOrca** [comment]
No because you have all the games from the bundle you downloaded from biorand.net. And they won't be marked read only.

**Comment by ashleycroft** [comment]
Okay, but i still dont know which RE 1 installation directory you are
referring to. I have the jpn sourcenext version installed.
